docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,328 @@
+    :align: center
+    :scale: 50
+    :alt: Control Center Settings Baud
+
+    Setting Baud Rate
+
+With the modules set to unique module IDs, and the baud rate set to match the flight controller's, you can now connect your modules to the flight controller. To do this, find the flight controller's serial port configured to run the Vertiq IO module, and connect the TX of the serial port to each module's RX port. Connect the RX of the serial port to each module's TX port. Connect a command ground between the flight controller and each module.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/module_wiring.png
+    :align: center
+    :scale: 100
+    :alt: Module Flight Control Wiring
+
+    Module Flight Control Wiring
+
+Integration Setup
+=================
+
+.. warning::
+    Please remove all propellers from any module you plan on testing. Failure to do so can result in harm to you or others around you. Further, please ensure that your module is secured to a stationary platform or surface before attempting to spin it. 
+
+Connect your flight controller and motors as shown in the diagram above. Ensure that both the motors and flight controller are receiving power.  :ref:`A short set of tones<startup_song>` should play from each motor when powered on. Connect the flight controller to QGroundControl, navigate to the :red:`parameters menu`, and select the :blue:`Vertiq IO` subsection as shown below.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/vertiq_parameters.png
+    :align: center
+    :scale: 50
+    :alt: PX4 Vertiq IO Parameters
+
+    PX4 Vertiq IO Parameters
+
+The parameters available in this subsection either control how PX4 talks to the modules, or parameters that are specific to the module. Parameters that are specific to the module are indicated by ‘Module Param -’ in their description.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/module_parameters.png
+    :align: center
+    :scale: 50
+    :alt: PX4 Module Parameters
+
+    PX4 Module Parameters
+
+The :blue:`TARGET_MODULE_ID` parameter sets which module the Module Params are synchronized with. When this parameter is written, the Module Params will be fetched from the module with that module ID and the values displayed for the Module Params will be updated. If you would like to specifically synchronize those parameters without switching module IDs you can set :red:`TRIGGER_READ`. It will immediately set itself back to disabled, but the Module Params will synchronize.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/trigger_read.png
+    :align: center
+    :scale: 50
+    :alt: PX4 Module Parameters
+
+    PX4 Module Parameters
+
+In this example we will be setting the module's input parser up in PWM mode. We will be setting up the motors to work as if the quadcopter is set up as in the diagram below.
```

Comment:
This is a comment from before, but it may also have gotten lost in the shuffle, reposting: 
Customers are never-endingly confused by this terminology, can you clarify here that this is the PWM mode of the motor's input parser, and is not related to the hobby protocol of PWM